java/geodesic-operations/README.md

```diff
@@ -1,36 +1,35 @@
-# Geodesic Operations 
-Use the GeometryEngine to calculate a geodesic path between two points and measure its distance.
+# Geodesic operations
 
-![Geodesic Operations App](geodesic-operations.png)
+Calculate a geodesic path between two points and measure its distance.
+
+![Image of geodesic operations](geodesic-operations.png)
+
+## Use case
+
+A geodesic distance provides an accurate, real-world distance between two points. Visualizing flight paths between cities is a common example of a geodesic operation since the flight path between two airports takes into account the curvature of the earth, rather than following the planar path between those points, which appears as a straight line on a projected map.
 
 ## How to use the sample
-Click on the map to select a path destination. The geodesic path between the two points will update.
+
+Tap anywhere on the map. A line graphic will display the geodesic line between the two points. In addition, text that indicates the geodesic distance between the two points will be updated. Tap elsewhere and a new line will be created.
 
 ## How it works
-1. Create a `Polyline` using two points.
-1. Pass this polyline to: `GeometryEngine.densifyGeodetic(polyline, segmentLength, unitOfMeasurement, GeodeticCurveType.GEODESIC)`. This will create a new polyline with segments of length `segmentLength` and `LinearUnit` set to `unitOfMeasurement`. The curve will be geodesic.
-1. You can set this geometry to a `Graphic` to display the curve in a `GraphicsOverlay`.
-1. To get the distance, use `GeometryEngine.lengthGeodetic(pathGeometry, unitOfMeasurement, GeodeticCurveType.GEODESIC)`  
-
-
-## Relevant API 
- * ArcGISMap 
- * GeodeticCurveType 
- * Geometry 
- * GeometryEngine 
- * Graphic 
- * GraphicsOverlay 
- * LinearUnit 
- * LinearUnitId 
- * MapView 
- * Point 
- * PointCollection 
- * Polyline 
- * SimpleLineSymbol 
- * SimpleMarkerSymbol
- 
-#### Tags
-Analysis
-Geometry
-Polyline
-GeometryEngine  
+
+1. Create a `Point` in New York City and display it as a `Graphic`.
```

Comment:
For how the sample works, we don't need to be super specific about where the point is since it's documented in the code comment
```suggestion
1. Create a `Point` and display it as a `Graphic`.
```